mne/time_frequency/tests/test_tfr.py

```diff
@@ -375,6 +376,25 @@ def test_plot():
     tfr.plot_topo(picks=[1, 2])
     plt.close('all')
 
+    fig = tfr.plot(picks=[1], cmap='interactive')  # test interactive cmap
+    fig.canvas.key_press_event('up')
+    fig.canvas.key_press_event('down')
+
+    cbar = fig.get_axes()[0].CB  # Fake dragging with mouse.
+    event = backend_bases.MouseEvent('button_press_event', fig.canvas, 0.1,
+                                     0.1, button=1)
+    event.inaxes = cbar.cbar.ax
+    cbar.on_press(event)
+    event.y = 0.2
+    cbar.on_motion(event)
```

Comment:
If it's because you need these, maybe we should add `_fake_*` wrappers for the rest, too. It seems like this should somehow be combined or integrated with the existing `_fake_` code